### PR TITLE
Update LICENSE and README

### PR DESCRIPTION
**Pull request by CHTJonas**
Housekeeping PR. This majorly rewords the README and adds the current year to the LICENSE file, together with a nod to other contributors.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Cameron Sparr
+Copyright (c) 2016–2020 Cameron Sparr and contributors
```
Comment (SuperQ) [inline LICENSE:3]: It's unnecessary to change the date. This just creates toil of having to update this date every year.

File: README.md [hunk 2]
```diff
@@ -50,59 +50,74 @@ fmt.Printf("PING %s (%s):\n", pinger.Addr(), pinger.IPAddr())
 pinger.Run()
 ```
 
-It sends ICMP packet(s) and waits for a response. If it receives a response,
-it calls the "receive" callback. When it's finished, it calls the "finish"
-callback.
+It sends ICMP Echo Request packet(s) and waits for an Echo Reply in
+response. If it receives a response, it calls the "receive" callback.
+When it's finished, it calls the "finish" callback.
```
Comment (SuperQ) [inline README.md:55]:
```suggestion
response. If it receives a response, it calls the "OnRecv" callback.
When it's finished, it calls the "OnFinish" callback.
```

**Review by SuperQ — COMMENTED** [review]
A couple of minor changes.

File: LICENSE (hunk 1 above)
Comment (CHTJonas) [inline LICENSE:3]: Good point. How's this instead?

```suggestion
Copyright (c) 2016 and later, Cameron Sparr and contributors.
```

Comment (CHTJonas) [reply LICENSE:3]: Good point. How's this instead?

```suggestion
Copyright (c) 2016 and later, Cameron Sparr and contributors.
```

Comment (SuperQ) [reply LICENSE:3]: It can just be the original date, iirc, "and later" is implied.

Comment (CHTJonas) [reply LICENSE]: Done! 🆗

**Review by SuperQ — APPROVED** [review]
LGTM

**Review by sparrc — APPROVED** [review]

**Pull request merged**